scripts/clean

```diff
@@ -3,3 +3,4 @@
 cd $(dirname $0)/..
 
 rm -rf CattleAgent.egg-info dist .tox
+find . -name *.pyc -exec rm -rf {} \;
```

Comment:
Would you want to add pyo to this just for the sake of completeness?
...This is a non-blocking suggestion. I don't care if we do or not.